quipucords/quipucords/settings.py

```diff
@@ -483,30 +483,36 @@ def app_secret_key_and_path():
 # the application will continue to work without the Celery scan manager enabled.
 # Once quipucords only supports the Celery Scan Manager and Redis, we can then
 # switch the default to be Redis.
+
+# When enabling the Celery Scan Manager, we want to also enable the Redis
+# Cache back-end. That can be disabled via QUIPUCORDS_ENABLE_REDIS_CACHE
+# for development or testing purposes.
+
+defaultEnableRedisCache = False
+if QPC_ENABLE_CELERY_SCAN_MANAGER:
+    defaultEnableRedisCache = True
+
+QUIPUCORDS_ENABLE_REDIS_CACHE = env.bool(
+    "QUIPUCORDS_ENABLE_REDIS_CACHE", defaultEnableRedisCache
+)
+
```

Comment:
mmmm we need more ruff checks. this is violating the style we adopted.

please use snake case for this variable.

alternatively, I think you could simply use `QPC_ENABLE_CELERY_SCAN_MANAGER` as the default value for `QUIPUCORDS_ENABLE_REDIS_CACHE`

```
QUIPUCORDS_ENABLE_REDIS_CACHE = env.bool("QUIPUCORDS_ENABLE_REDIS_CACHE", QPC_ENABLE_CELERY_SCAN_MANAGER)
```